### ISSUE TITLE
DiffPure purification method integrated

### ISSUE THREAD

**Issue by rathoreyuvraj** [PolyLiYJ/StyleGuard#2]
Hi, Really appreciate your work. I had some questions:
1. May I know is there any plan to release DiffPure purifier integrated StyleGuard training code.
2. I was trying to map the loss functions mentioned in the paper there I couldn't find L(Denoise) in the StyleGuard code. Can you help me locate and understand it.

Thanks

**Comment by PolyLiYJ** [comment]
Hi，
we will publish the diffpure code in next week. For the denoising loss，it
is the same  with the Upscaling loss. The only difference is the model to
compute the loss. It is computed in Line 709 in styleguard.py
Best Regards
Yanjie


Yuvraj singh rathore ***@***.***> 于 2025年12月17日周三 下午7:52写道：
